config.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Theres no need to remove tests, I still want to be able to run cart pole.  Just add yours.